app/presenters/idv/welcome_presenter.rb

```diff
@@ -0,0 +1,81 @@
+module Idv
+  class WelcomePresenter
+    include ActionView::Helpers::TranslationHelper
+    include Rails.application.routes.url_helpers
+    include LinkHelper
+    include ActionView::Helpers::UrlHelper
+
+    attr_accessor :url_options
+
+    def initialize(sp_session)
```

Comment:
`sp_session` is a method in the controller that is expected to be a hash. Wdyt of changing this arg to `decorated_sp_session`? I was a touch confused when I saw us called `#selfie_required?` and `#sp_name` on what I had assumed would be a hash.

Reply:
Renamed.